empress/recon_vis/tanglegram.py

```diff
@@ -0,0 +1,141 @@
+"""
+tanglegram.py
+Visualizes tanglegrams using matplotlib
+Berto Garcia, Sonia Sehra
+"""
+
+from empress.recon_vis import utils, plot_tools, tree
+
+VERTICAL_OFFSET = 20
+HORIZONTAL_SPACING = 10
+LEAF_SPACING = 5
+HOST_COUNTER = 0
```

Comment:
This is not a constant. Please use a naming convention that indicates that it is not a constant.

Reply:
For example `g_host_counter`, `g_parasite_counter` where `g` stands for global. You might also want to keep this variable internal to this file. We could write `_g_host_counter`, `_g_parasite_counter`.

Reply:
Also note that using global variables is discouraged in all languages I know of.